configs/animatediff/README.md

```diff
@@ -0,0 +1,204 @@
+
+This model has several weights including vae, unet and clip. You should download the weights from [stable-diffusion-1.5](https://huggingface.co/runwayml/stable-diffusion-v1-5) and change the 'pretrained_model_path' in config to the weights dir.
+
+|                           Model                           | Dataset |                                     Download                                      |
+| :-------------------------------------------------------: | :-----: | :-------------------------------------------------------------------------------: |
+|            [ToonYou](./animatediff_ToonYou.py)            |    -    |              [model](https://civitai.com/api/download/models/78775)               |
+|             [Lyriel](./animatediff_Lyriel.py)             |    -    |              [model](https://civitai.com/api/download/models/72396)               |
+|        [RcnzCartoon](./animatediff_RcnzCartoon.py)        |    -    |              [model](https://civitai.com/api/download/models/71009)               |
+|           [MajicMix](./animatediff_MajicMix.py)           |    -    |              [model](https://civitai.com/api/download/models/79068)               |
+|    [RealisticVision](./animatediff_RealisticVision.py)    |    -    |              [model](https://civitai.com/api/download/models/29460)               |
+| [RealisticVision_v2](./animatediff_RealisticVision_v2.py) |    -    | [model](https://huggingface.co/guoyww/animatediff/resolve/main/mm_sd_v15_v2.ckpt) |
+
+## Quick Start
+
+Running the following codes, you can get a text-generated image.
+
+1. Download [ToonYou](https://civitai.com/api/download/models/78775) and MotionModule checkpoint
+
+```bash
+#!/bin/bash
+
+gdown 1RqkQuGPaCO5sGZ6V6KZ-jUWmsRu48Kdq -O models/Motion_Module/
+gdown 1ql0g_Ys4UCz2RnokYlBjyOYPbttbIpbu -O models/Motion_Module/
+wget https://civitai.com/api/download/models/78775 -P models/DreamBooth_LoRA/ --content-disposition --no-check-certificate
+```
+
+2. Modify the config file in `configs/animatediff/animatediff_ToonYou.py`
+
+```python
+
+    motion_module_cfg=dict(
+        path={Your MotionModule path}
+    ),
+    dream_booth_lora_cfg=dict(
+        type='ToonYou',
+        path={Your Dreambooth_Lora path},
+        steps=25,
+        guidance_scale=7.5))
+```
+
+3. Enjoy Text2Video world
+
+```python
+import cv2
+import numpy as np
+import mmcv
+from mmengine import Config
+from PIL import Image
+
+from mmagic.registry import MODELS
+from mmagic.utils import register_all_modules
+
+import os
+import torch
+from pathlib import Path
+import datetime
+from mmagic.models.editors.animatediff import save_videos_grid
+
+
+
+register_all_modules()
+
+cfg = Config.fromfile('configs/animatediff/animatediff_ToonYou.py')
+animatediff = MODELS.build(cfg.model).cuda()
```

Comment:
![image](https://github.com/open-mmlab/mmagic/assets/12782558/06c69c81-5e31-4944-ae98-375a865c57b1)

when I run these codes, I get this error above.
this is because If  'runwayml/stable-diffusion-v1-5' weights downloaded from huggingface automaticly, there is no diffusion_pytorch_model.bin (only diffusion_pytorch_model.safetensors). So you should add some codes to make it work with safetensors.

Reply:
I've added automatic weights downloading and successfully loaded safetensors.